### ISSUE TITLE
Cannot get GraphiQL to send/display requests

### ISSUE THREAD

**Issue by sulliwane** [graphql/express-graphql#34]
I installed express-graphql, when I open **112.74.106.164:3000/graphql** in the browser, graphiql is loaded. But nothing happen on graphql request. For example:
```
{
  authors {
    name
  }
}
```
Mongodb is running well, and this request work if I run directly from the server. But no chance through Graphiql...

Here is my **server.js**:
```
require('babel/register');
var express = require('express');
var graphql = require('graphql');
var expressGraphql = require('express-graphql');
var Schema = require('./server/schema.js');

var app = express();

app.use('/', expressGraphql({
  schema: Schema,
  graphiql: true
}));

app.listen(3000);
console.log('GraphQL Sandbox started on port: 3000');
```

And here is the **schema.js**:
```
const mongo = require('promised-mongo');
const db = mongo('mongodb://localhost/mydb');
const authorsCollection = db.collection('authors');

import * as _ from 'underscore';

import {
  GraphQLList,
  GraphQLObjectType,
  GraphQLSchema,
  GraphQLString,
  GraphQLInt,
  GraphQLFloat,
  GraphQLEnumType,
  GraphQLNonNull
} from 'graphql';

const Author = new GraphQLObjectType({
  name: 'Author',
  description: 'Represent the type of an author of a blog post or a comment',
  fields: () => ({
    _id: {type: GraphQLString},
    name: {type: GraphQLString},
    twitterHandle: {type: GraphQLString}
  })
});

const Query = new GraphQLObjectType({
  name: 'RootQuery',
  fields: {
    authors: {
      type: new GraphQLList(Author),
      resolve: function() {
        return authorsCollection.find().toArray();
      }
    }
  }
});
const Mutation = new GraphQLObjectType({
  name: 'Mutations',
  fields: {
    createAuthor: {
      type: Author,
      args: { 
        _id: {type: new GraphQLNonNull(GraphQLString)},
        name: {type: new GraphQLNonNull(GraphQLString)},
        twitterHandle: {type: GraphQLString}
      },
      resolve: function(rootValue, args) {
        let author = _.clone(args);
        return authorsCollection.insert(author)
          .then(_ => author);
      }
    }
  }
});

const Schema = new GraphQLSchema({
  query: Query,
  mutation: Mutation
}); 

export default Schema;
```

Any help would be much appreciated, Thanks :)

**Comment by sulliwane** [comment]
I believe it's a problem with firefox, as after switching to Opera or Chrome I could see GraphiQL responses.

**Comment by leebyron** [comment]
This issue will be fixed by v0.4.5 now on npm
